src/common/OptionsMenu/ValidationTemplate/ValidationTemplate.module.scss

```diff
@@ -0,0 +1,27 @@
+.validation_option {
```

Comment:
We usually use BEM methodology for naming classes.
Two underscores are used for `element`
One underscore is used for `modificator`
In this case it should be `validation-option`
Inner classes must be renames as well. For icon `validation-option__icon`, and for validity `validation-option__icon_valid` and `validation-option__icon_invalid`
If you prefer not to use BEM, then it's better not to use underscores in class names